publish.gradle

```diff
@@ -103,6 +103,15 @@ ext.configurePublish = { jnaForTestConfiguration = null ->
                         }
                     }
 
+                    def depLicenses = new XmlSlurper().parse(new File("${projectDir}/dependency-licenses.xml"))
```

Comment:
NEAT! So this gets added to our artefacts. Nice.

Reply:
Yes, copied over from a-s :D